pkg/rpc/request/param.go

```diff
@@ -240,25 +303,17 @@ func (p *Param) GetBytesBase64() ([]byte, error) {
 
 // GetSignerWithWitness returns SignerWithWitness value of the parameter.
 func (p *Param) GetSignerWithWitness() (SignerWithWitness, error) {
-	c, ok := p.Value.(SignerWithWitness)
-	if ok {
-		return c, nil
-	}
-	raw, ok := p.Value.(json.RawMessage)
-	if !ok {
-		return SignerWithWitness{}, errors.New("not a signer")
-	}
 	aux := new(signerWithWitnessAux)
-	err := json.Unmarshal(raw, aux)
+	err := json.Unmarshal(p.RawMessage, aux)
 	if err != nil {
-		return SignerWithWitness{}, errors.New("not a signer")
+		return SignerWithWitness{}, fmt.Errorf("not a signer: %w", err)
 	}
-	accParam := Param{StringT, aux.Account}
+	accParam := Param{RawMessage: []byte(fmt.Sprintf(`"%s"`, aux.Account))}
```

Comment:
We can avoid calling `fmt.Sprintf` and creating `Param` here, we already have the string.